### PR TITLE
Move all quarantined tests to the aspnetcore-quarantined-tests pipeline

### PR DESCRIPTION
**Pull request by BrennanConroy**
Example https://dev.azure.com/dnceng/public/_build/results?buildId=667381&view=results

### CODE REVIEW

File: .azure/pipelines/quarantined-tests.yml [hunk 1]
```diff
@@ -46,3 +46,103 @@ jobs:
...
+      condition: always()
+    - task: PublishTestResults@2
+      displayName: Publish Quarantined Test Results
+      inputs:
+        testResultsFormat: 'xUnit'
+        testResultsFiles: '*.xml'
+        searchFolder: '$(Build.SourcesDirectory)/artifacts/TestResults/$(_BuildConfig)/Quarantined'
+      condition: always()
+    artifacts:
+    - name: Windows_Quarantine_Test_Logs
```
Comment (dougbu) [inline .azure/pipelines/quarantined-tests.yml:74]: Suggest using "Quarantined" consistently

**Review by dougbu — APPROVED** [review]
Looks good. My questions / comments are all suggestions or nits except the ones about defaults and the extra steps for restore operations.

File: src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs [hunk 2]
```diff
@@ -250,10 +250,21 @@ private void ValidateAppWorksOffline(Project project, string listeningUri, bool
             TestBasicNavigation(project.ProjectName, skipFetchData: skipFetchData);
         }
 
-        [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public async Task BlazorWasmHostedTemplate_IndividualAuth_Works(bool useLocalDb)
+        [ConditionalFact]
+        // LocalDB doesn't work on non Windows platforms
```
Comment (pranavkm) [inline src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs:254]: cc @javiercn

Comment (javiercn) [reply src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs:254]: Yeah, if we have to break them into two separate tests, so be it

**Pull request merged**